src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,125 @@
-package co.inventorsoft.academy.collections.model;
+package main.java.co.inventorsoft.academy.collections.model;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
+import java.util.function.Function;
 
 public class Range<T> implements Set<T> {
```

Comment:
Implement this part of the task " **Custom types can be added via implementation of Comparable**